provider/runbook_resource.go

```diff
@@ -96,6 +96,16 @@ func resourceRunbook() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			"attachment_rule": {
+				Default:          "null",
+				Type:             schema.TypeString,
```

Comment:
i think with a little tweaking of the logic where we set this value in the read function, we can avoid needing to do a Default: null here
```suggestion
			"attachment_rule": {
				Type:             schema.TypeString,
```